drivers/py/pes/metatensor.py

```diff
@@ -0,0 +1,43 @@
+"""Interface with metatensor to run machine learning potentials"""
```

Comment:
I would add a link to something like https://lab-cosmo.github.io/metatensor/latest/atomistic/index.html here